man/signal-cli.1.adoc

```diff
@@ -252,6 +252,10 @@ Specify the contact phone number.
 *-n*, *--name*::
 Specify the new name for this contact.
 
+*-e*, *--expiration*::
+Set expiration time of messages (seconds).
+By default expiration is set to 0, no expiration.
```

Comment:
hm, this is a bit misleading, when adding a new contact, the expiration is set to 0 by default, but when updating a contact the expiration is left unchanged by default.

Reply:
I changed the second part to `To disable expiration set expiration time to 0.`, does it feel better? Or would you remove that part completely?